### PR TITLE
feat(#427): add age

### LINKED ISSUE #427: feat: add age

### PR DESCRIPTION
**Pull request by HLasse**
Very open for comments. Left the naming very open (in case there is a use for getting the time from some event), but might be better to rename to `AgeSpec` or something. Not happy about having to specify the output colname in the spec as we don't do that in other places, but unsure what the best alternative approach is if we want to keep it open for use cases besides age (where it would be easier to hardcode).

feat(#427): add age

Fixes #427

feat: age / time from event spec

### CODE REVIEW

**Comment by HLasse** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #459** <a href="https://app.graphite.dev/github/pr/Aarhus-Psychiatry-Research/timeseriesflattener/459?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Aarhus-Psychiatry-Research/timeseriesflattener/459?utm_source=stack-comment).

File: src/timeseriesflattenerv2/_process_spec.py [hunk 1]
```diff
@@ -152,3 +159,38 @@ def process_spec(
         ),
         pred_time_uuid_col_name=predictiontime_frame.pred_time_uuid_col_name,
     )
+
+
+def process_time_from_event_spec(
```
Comment (MartinBernstorff) [inline src/timeseriesflattenerv2/_process_spec.py:164]: Isn't currently being called by Flattener.

I think having three paths (StaticSpec, TimeDeltaSpec and TemporalSpec) is a good move for the top level function here 👍

File: src/timeseriesflattenerv2/feature_specs.py [hunk 2]
```diff
@@ -232,6 +232,31 @@ def df(self) -> pl.LazyFrame:
...
 
 
+@dataclass
+class TimeFromEventSpec:
+    init_frame: TimestampValueFrame
+    fallback: ValueType
+    output_name: str
+    column_prefix: str = "pred"
+
+    def __post_init__(self):
```
Comment (MartinBernstorff) [inline src/timeseriesflattenerv2/feature_specs.py:242]: Test that this errors correctly?

File: src/timeseriesflattenerv2/_process_spec.py [hunk 3]
```diff
@@ -8,10 +8,14 @@
...
 from .feature_specs import (
     Aggregator,
+    BooleanOutcomeSpec,
     LookPeriod,
+    OutcomeSpec,
     PredictionTimeFrame,
+    PredictorSpec,
     ProcessedFrame,
     TimedeltaFrame,
+    TimeFromEventSpec,
```
Comment (MartinBernstorff) [inline src/timeseriesflattenerv2/_process_spec.py:18]: TimedeltaSpec?

**Pull request merged**